db/scripts/dump.sql

```diff
@@ -0,0 +1,42 @@
+CREATE DATABASE IF NOT EXISTS AuctionDB;
+
+CREATE TABLE IF NOT EXISTS `AuctionDB`.`USERS` (
+`ID` BIGINT NOT NULL AUTO_INCREMENT,
+`NAME` VARCHAR(255) NULL,
+`PASSWORD` BINARY(60) NULL,
+PRIMARY KEY (`ID`),
+UNIQUE KEY (`NAME`),
+UNIQUE KEY (`PASSWORD`))
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE IF NOT EXISTS `AuctionDB`.`AUCTIONS` (
+`ID` BIGINT NOT NULL AUTO_INCREMENT,
+`TITLE` VARCHAR(255) NULL,
```

Comment:
I'm not sure is it a good idea to allow user to type tittle name for 255 symbols